mbq/metrics/__init__.py

```diff
@@ -13,31 +13,32 @@
 logger = logging.getLogger('mbq.metrics')
 
 _initialized = False
+_namespace = None
+_constant_tags = {}
 _statsd = utils.NullStatsd()
 
 
 def init(namespace=None, constant_tags=None):
-    global _initialized, _statsd
+    global _initialized, _namespace, _constant_tags, _statsd
     if _initialized:
         logger.warning('mbq.metrics already initialized. Ignoring re-init.')
         return
 
     if constant_tags:
-        constant_tags = utils.tag_dict_to_list(constant_tags)
+        _constant_tags = constant_tags
 
+    _namespace = namespace
     _statsd = datadog.DogStatsd(
```

Comment:
Now that we don't pass anything interesting to the DogStatsd I think this can happen outside of the `init` function.

Reply:
yeah so I tried that, but it made the tests fail. I think we don't want to initialize _statsd on import we can mock it out.